### ISSUE TITLE
Duplicated entries in FOCUSOBE

### ISSUE THREAD

**Issue by monodera** [naojsoft/qplan#7]
In the OPE file generated by `qplan`, `INSROT_PA` (and `Filter` and `FILTER`?) appears twice. 

```
#FOCUSOBE $DEF_IMAGE $T15_HETDEX_012 INSROT_PA=0.0 OFFSET_RA=0 OFFSET_DEC=0 Filter="HSC-r2" INSROT_PA=0.00 DELTA_Z=0.05 DELTA_DEC=5 FILTER="HSC-r2" EXPTIME=10 Z=3.75
```

Thanks.

**Comment by ejeschke** [comment]
@monodera, in the template for generating the command, the `INSROT_PA` and `FILTER` appear only once.  I think if they are also defined in `$DEF_IMAGE` (once in integgui2) they will appear twice.  But there is no knowledge of the contents of `$DEF_IMAGE` at the time of generating the OPE content from qplan.

**Comment by ejeschke** [comment]
Basically, can you force it to appear twice when generating the OPE strictly in qplan?

**Comment by monodera** [comment]
Only with qplan, it always appears twice except for the very first one outiside of any Start_OB/Stop_OB pairs.

**Comment by ejeschke** [comment]
This is fixed in commit acb989f009c7e096c8a96ba3da16e8841bf9350b
